src/app.js

```diff
@@ -2,11 +2,13 @@
 
 import { quizData } from './data.js';
 import { initWelcomePage } from './pages/welcomePage.js';
+import { drawText } from './pages/canvasPage.js';
 
 const loadApp = () => {
   quizData.currentQuestionIndex = 0;
 
   initWelcomePage();
+  setInterval(drawText, 50);
```

Comment:
Instead of `setInterval` look into `requestAnimationFrame`. That will help performance (a bit): https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame

Reply:
i suppose to use `requestAnimationFrame` but i dont have flexibility to add timer to draw so for that i use `setInterval`but if you have idea if i can change time of drawing using `requestAnimationFrame`  i will be so happy to know how

Reply:
I see your point, the animation is kind of depending on the speed